appeals/web/src/server/lib/mappers/appeal/fpa.js

```diff
@@ -0,0 +1,16 @@
+import { submaps as hasSubmaps } from './has.js';
+import { mapAppellantFinalCommentDueDate } from './submappers/appellant-final-comment-due-date.mapper.js';
+import { mapIpCommentsDueDate } from './submappers/ip-comments-due-date.mapper.js';
+import { mapLpaFinalCommentDueDate } from './submappers/lpa-final-comment-due-date.mapper.js';
+import { mapLpaStatementDueDate } from './submappers/lpa-statement-due-date.mapper.js';
+import { mapS106ObligationDue } from './submappers/s106-obligation-due-date.mapper.js';
+
+/** @type {Record<string, import('./appeal.mapper.js').SubMapper>} */
+export const submaps = {
```

Comment:
Very pedantic point, and not worth blocking approval, but I'm still not keen on this naming and don't see why this can't be called `submappers`, which is not much longer but much more descriptive imo.